### PR TITLE
Enhancement 8 daniel reconfigured database and seed

### LINKED ISSUE #8: Car Model and Seeds
Car Model
* `make` (str)
* `model`  (str)
* `year` (int)
*  `max_range` (int)

In `db/seeds.rb`, make an entry for every car you want to support.

### PR DESCRIPTION
**Pull request by Daniel-McGlinchy**
closes #8

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_05_26_172042) do
+
+  create_table "vehicles", force: :cascade do |t|
+    t.string "make"
+    t.string "name"
+    t.integer "range"
```
Comment (andyweiss1982) [inline db/schema.rb]: Is the “name” field akin to “model”?

What about year?  Does it matter, or do the ranges not vary from year to year?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,22 @@
...
+
+response = HTTParty.get('https://evbite.com/all-electric-cars-available-in-2018/'); nil
+parsed_body = Nokogiri::HTML(response.body); nil
+vehicles_table = parsed_body.css('#tablepress-8 tbody tr'); nil
+
+vehicles = vehicles_table.map do |vehicle|
+  {
+      make: vehicle.css('.column-1').text.downcase,
+      name: vehicle.css('.column-2').text,
+      range: vehicle.css('.column-4').text.split(' ')[0]
```
Comment (andyweiss1982) [inline db/seeds.rb]: Is there a max / min range? 

Does it matter which one you pick?

File: db/schema.rb (hunk 1 above)
Comment (Daniel-McGlinchy) [reply db/schema.rb]: Yes, the name is model. That was a hold over from the setup in the other PR.

The site we got the info from doesn't list the year. These are all the electric vehicles as of 2018. Each model is listed once. A few models had a min and max integer for range. This is likely the improvement in range over model years. We went for the min value to be on the safe side.

File: db/seeds.rb (hunk 2 above)
Comment (Daniel-McGlinchy) [reply db/seeds.rb]: A few models had a min and max integer for range. This is likely the improvement in range over model years. We went for the min value to be on the safe side.

**Comment by Daniel-McGlinchy** [comment]
We were originally concerned with the duplication of information in our selectors. Nick suggested that creating two tables would solve that. That is why we went with the two table ERD in the other PR. Tommy walked us through how we could use `pluck` and `.uniq` for example to achieve the same result while only having one table.

**Comment by Daniel-McGlinchy** [comment]
> This looks good to me. I would still consider changing “name” to “model” but it’s not a dealbreaker. I will approve for now but let’s still hold off on merging until we are all looking at the same computer.
> 
> Good job

Awesome! We will finish up our work on the other issue tomorrow and make a PR for it. If all goes well we will be able to have both issues cleared on Tuesday.

File: db/seeds.rb (hunk 2 above)
Comment (andyweiss1982) [reply db/seeds.rb]: ok makes sense

File: db/schema.rb (hunk 1 above)
Comment (andyweiss1982) [reply db/schema.rb]: OK I am fine with this

**Pull request merged**